src/components/Comments/ContainerComment.js

```diff
@@ -34,7 +34,8 @@ export const ReplyComment = ({
   updateVote,
   feedId,
   level = 1,
-  onReplyButtonClick
+  onReplyButtonClick,
+  refreshComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refreshComment` prop is a good approach to handle the refresh logic. It seems like this prop will be used to trigger a refresh in the comment section whenever a user action requires it.
```diff
+  refreshComment
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->